lib/manageiq/ui/classic/engine.rb

```diff
@@ -28,6 +29,13 @@ module Classic
       class Engine < ::Rails::Engine
         config.autoload_paths << File.expand_path(File.join(root, 'app', 'controllers', 'mixins'), __FILE__)
         config.autoload_paths << File.expand_path(File.join(root, 'lib'), __FILE__)
+        config.assets.js_compressor = Uglifier.new(
+          :compress => {
+            :unused      => false,
+            :keep_fargs  => true,
+            :keep_fnames => true
+          }
+        ) if Rails.env.production?
```

Comment:
I couldn't find any examples of doing engine configuration changes in production mode only... is this the best way?